### PR TITLE
Create project structure for Core SDK

### PR DESCRIPTION
**Pull request by kpanwar**
Create project structure for Core SDK, per #916

### CODE REVIEW

File: tests/core/Test.Microsoft.Identity.Core.Android/Test.Microsoft.Identity.Core.Android.csproj [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    <AndroidLinkMode>SdkOnly</AndroidLinkMode>
+    <EmbedAssembliesIntoApk>True</EmbedAssembliesIntoApk>
+  </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="Mono.Android" />
+    <Reference Include="mscorlib" />
+    <Reference Include="System" />
+    <Reference Include="System.Core" />
+    <Reference Include="System.Xml" />
+    <Reference Include="Xamarin.Android.NUnitLite" />
```
Comment (jmprieur) [inline tests/core/Test.Microsoft.Identity.Core.Android/Test.Microsoft.Identity.Core.Android.csproj]: These tests don't seem to appear in the IDE

File: tests/core/Test.Microsoft.Identity.Core.Android/Test.Microsoft.Identity.Core.Android.csproj [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="MainActivity.cs" />
+    <Compile Include="TestSample.cs" />
```
Comment (jmprieur) [inline tests/core/Test.Microsoft.Identity.Core.Android/Test.Microsoft.Identity.Core.Android.csproj]: These tests don't seem to appear in the IDE

File: tests/core/Test.Microsoft.Identity.Core.Android/Test.Microsoft.Identity.Core.Android.csproj (hunk 1 above)
Comment (kpanwar) [reply tests/core/Test.Microsoft.Identity.Core.Android/Test.Microsoft.Identity.Core.Android.csproj]: I will merge dev into my branch.

File: tests/core/Test.Microsoft.Identity.Core.Android/Test.Microsoft.Identity.Core.Android.csproj (hunk 2 above)
Comment (kpanwar) [reply tests/core/Test.Microsoft.Identity.Core.Android/Test.Microsoft.Identity.Core.Android.csproj]: there are no tests written. this is just an empty skeleton project

**Pull request merged**